README.md

```diff
@@ -33,15 +35,28 @@ Coming soon.
 
 ## Running the site locally
 
-If you want to develop the site locally, you can run a single command:
+If you want to develop the site locally, you can run a single command (assuming that you've run the [setup](#setup)):
 
 ```bash
 $ make dev
 ```
 
-## Contributing
+This will start up a local server on localhost port 1313. When you make changes to either the content of the website (in [`content`](content)) *or* to the Sass and JavaScript assets of the page (in [`themes/jaeger-docs/source`](themes/jaeger-docs/source)), the browser will automatically update to reflect those changes (usually in under one second).
+
+## Publishing the site
+
+The site is published automatically by [Netlify](https://www.netlify.com/) whenever changes are merged to the `master` branch. The site cannot be published in an ad-hoc way (e.g. through a `make` command or script in the repo).
+
+## Contributing to the site
+
+We strongly encourage you to contribute to this site!! There are several ways you can do so:
+
+* By filing [issues](issues) if you see errors, would like to request content, etc.
+* By submitting [pull requests](pulls) with desired changes
+
+If you submit a pull request, there are two things to be aware of:
```

Comment:
I think the above addition to Contributing section is good, but don't remove the link to the full CONTRIBUTING doc, which has specific instructions.

Reply:
BTW, this is just a WIP